react/features/base/avatar/components/web/Avatar.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import React from 'react';
+
+import { connect } from '../../../redux';
+
+import AbstractAvatar, {
+    _mapStateToProps,
+    type Props as AbstractProps,
+    DEFAULT_SIZE
+} from '../AbstractAvatar';
+
+type Props = AbstractProps & {
+    className?: string,
+    id: string
+};
+
+/**
+ * Implements an avatar as a React/Web {@link Component}.
+ */
+class Avatar extends AbstractAvatar<Props> {
+
+    /**
+     * Constructs a style object to be used on the avatars.
+     *
+     * @param {string?} color - The desired background color.
+     * @returns {Object}
+     */
+    _getAvatarStyle(color) {
+        const { size = DEFAULT_SIZE } = this.props;
+
+        return {
+            backgroundColor: color || undefined,
```

Comment:
Why is the background color optional?

Reply:
Because then we use the default one that is defined in the class